### PR TITLE
Reduce elf-diffs done in CI, add a bdshot board

### PR DESCRIPTION
**Pull request by peterbarker**

### CODE REVIEW

**Comment by IamPete1** [comment]
I would be happy with no elf-diff if we could upload the with and without elf as artifacts making it much easier to run locally.

**Comment by peterbarker** [comment]
> I would be happy with no elf-diff if we could upload the with and without elf as artifacts making it much easier to run locally.

Is this something you use often?  tridge and I thought it was essentially unused, thus this PR.  If this is something you're using, we'll just drop this PR.

Note that there's `Tools/scripts/size_compare_branches.py` which allows you to do the same thing locally rather conveniently.

**Comment by IamPete1** [comment]
I have been looking at it ocationaly to understand unexpected size changes. Since we're still building both for the size change uploading the elf should not take much time. We could also upload all builds, currently we only run the diff on copter and plane I think.

I do think it's worth the actions time saving, without having to build just running the diff locally is probably not so much different than downloading and unzipping twice.

File: .github/workflows/test_size.yml [hunk 1]
```diff
@@ -22,6 +22,7 @@ jobs:
             Durandal,
             MatekF405,
             Pixhawk1-1M,
+            SkystarsH7HD-bdshot,
```
Comment (andyp1per) [inline .github/workflows/test_size.yml:25]: I would probably pick KakuteF7Mini so that we have F7 covered + flash

**Comment by peterbarker** [comment]
Closing this because it turns out people *do* find it useful :-)

**Pull request closed without merging**